ch04packaging/03Packaging.ipynb.py

```diff
@@ -21,43 +21,57 @@
 # A good cross-platform build tool is the most important thing: you can always
 # have collaborators build from source.
 #
+# Along with the content below, the Python ecosystem has a few fantastic guides on both simple
+# and complex packaging. Some of the recommended ones are listed below -
+#  - [Python Packaging User Guide](https://packaging.python.org/en/latest/)
+#  - [pyOpenSci Python Package Guide](https://www.pyopensci.org/python-package-guide/)
+#  - [Scientific Python Packaging Guide]https://learn.scientific-python.org/development/tutorials/packaging/
+#
+# Besides the guides, there are numerous "cookie" templates available for developers. These templates
+# allow developers to generate an empty Python package following good practices and guidelines with a
+# simple CLI command.
+#  - [Scientific Python cookie](https://github.com/scientific-python/cookie)
+#  - [Open Science Labs' scicookie](https://github.com/osl-incubator/scicookie)
+#  - Domain/ecosystem specific cookies like [pybamm-cookiecutter](https://github.com/pybamm-team/pybamm-cookiecutter) for battery modeling projects in Python exist too
 
 # %% [markdown]
 # ### Distribution tools
 
 # %% [markdown]
 # Distribution tools allow one to obtain a working copy of someone else's package.
 #
-# - Language-specific tools: 
-#  - python: PyPI,
-#  - ruby: Ruby Gems, 
-#  - perl: CPAN,
-#  - R: CRAN
+#  - Language-specific tools: 
+#    - python: PyPI,
+#    - ruby: Ruby Gems, 
+#    - perl: CPAN,
+#    - R: CRAN
```

Comment:
Maybe we should add here some explanation and links about this. In fact, I don't know whether our intention here was to link tools (`pip`, `gem`, ...) or repositories (which all are!). So maybe we could add a section about repositories (and mention that list together with the one below without much detail), and then there mentioning the tools to query them.

Reply:
Also probably I'd put higher up: python, julia, r and rust before perl or ruby nowadays.